### ISSUE TITLE
falta de aviso de ligar al alma

### ISSUE THREAD

**Issue by 50megas** [RoDNet/Bugtracker-Infinium-4.0.6a#459]
por fin consegui la http://es.wowhead.com/item=64377 ligada a cuenta y que puedes pasar a otro pj,me la pase al gerrero que en principio era para el,me la equipe por si las moscas y entoces mela made al dk para ver si podia usarla con los 2 pj y la estube probando la encante con enchand de dk ,luego fui a pasarla al gerrero y no me dejo, decia que estaba ligada al alma,cuando ami en nigun mometo me salio la advertencia de que se ligaba al alma.

siempre q vas encatar un odjeto que se liga ejemplo: unos brazales que se ligan al equipar y los tienes en la mochila y le quieres metes un encatamiento de profesion como peleteria te advierte antes, que al hacerlo se ligaran al pj ,igual con con piezas pvp que tieenen menos de 2h de haverlas comprado,siempre que las engemes o encantes perderas la 2h para devolverla y se ligaran ,pero te havisa.

comentado el caso con un gm ,me comento que esas armas, ponen en la descripcion que se ligan al equiparlas cosa que tampoco entiendo ya que la tube en 2 pj equipada y no salio tampoco ningun aviso

un saludo

**Comment by asfo** [comment]
Recuerda, no lo estás ligando de manera "de encantamiento", lo estás ligando con un hechizo especial de clase...Analiza que la diferencia es que si LEES, dice que es para DKs nada más, si es que estoy pensando en lo mismo que creo que hiciste...

QUIZÁ si le quitas el hechizo se pueda..Eso pasa cuando no entiendes lo que a hechizos DE CLASE se refiere..

**Comment by ghost** [comment]
Si, a mi me pasó al ponerle la cadena de pirium para pvp para el desarme, que se la fui a pasar y como la cadena dice que te liga el objeto a tu alma, no me dejaba pasársela a ninguno. Sin embargo si pones cualquier otro enchant, como por ejemplo Avalancha que sale rentable económicamente, ya se lo tendrías que poder pasar ;)

**Comment by 50megas** [comment]
si entiendo que se lige con un hechizo de clase o encatamiento o otra cosa ,lo que no entiendo es por que no sale un mesajito en el centro de la pantalla diciendo: si continuas este odjeto se ligara a ti  con la opciones de aceptar o cancelar

**Comment by 50megas** [comment]
zaska eso me estraña mas , entederia lo del encatamiento del dk pero eso,en el cata oficial por esperiencia propia yo encante http://es.wowhead.com/item=42952 un pieza ligada a cuenta y la encate con http://es.wowhead.com/spell=86375 un encatamieto de profesion que liga el odjeto al alma y podia pasarla a cualquier pj sin problema,lo unico que el encatamieto se ponia en rojo y ponia se necesita iscripcion y no valia con ese pj solo con iscriptores

te salio el aviso? de que se ligaria para aceptar o cancelar

**Comment by ghost** [comment]
No, no me salió el aviso para aceptar o cancelar, directamente puse http://es.wowhead.com/item=55057 sin aviso de ningún tipo, y luego ya no me dejó pasársela a ningún pj hasta que le puse un enchant normal y corriente

**Comment by 50megas** [comment]
parece que el bug noes que no salga el aviso,por que realmete no se liga al alma (ya que si estuviera ligado,por muchos enchad que le cambiaras no podrias pasarlo)

el bug viene a ser que al encatar una pieza ligada a cuenta, con algo que liga al alma no te deja embiarlo a otro pj dicendo que esta ligado al alma por el encatamiento.
cuando por esperiencia ,si se puede lo unico que los enchad no funciona con otros pj que no tengan los requisitos

ejemplo: el enchand de iscricion se pone en rojo y pide un minimo de abilidad (no recuendo exacto) de iscripcion y deja de funcionar hasta que se cumplan los requisitos.

asi deveria de pasar con todos . luego intentare testear con las hombreras que use en el oficial

**Comment by 50megas** [comment]
lo testee 

aqui la armadura con el encatamiento de iscripcion que liga las hombreras al alma

<a href="http://tinypic.com?ref=qzog9h" target="_blank"><img src="http://i50.tinypic.com/qzog9h.jpg" border="0" alt="Image and video hosting by TinyPic"></a>

aqui lo que sale al intentar envialo:

<a href="http://tinypic.com?ref=drb148" target="_blank"><img src="http://i50.tinypic.com/drb148.jpg" border="0" alt="Image and video hosting by TinyPic"></a>

esto en el cata oficial no pasaba,lo puedes enviar sin problema,pero pedira unos requisitos para que funcione

aqui el moje que me estube subiendo cuando me compre el pandariam con la hombreras encantadas y sin tener iscripcion, lo unico que en la armeria no sale en rojo pidiendo requisitos:

http://eu.battle.net/wow/es/character/uldum/Pöchii/simple

un saludo

**Comment by asfo** [comment]
Claro que no, en el oficial no podías enviar los items con encantamientos que ligaban el objeto así igual, siempre que mandabas un item heirloom con un encantamiento, este, desaparecía, claro, podías enviarlo, pero simplemente saltaba "Si envías este objeto, se eliminará el encantamiento" o algún mensaje así.

Aún así, investigando el DK si le aplica un encantamiento especial, hace que el objeto pierda la posibilidad de enviarse a menos que se le remueva el encantamiento.
